crates/lexarg/src/lib.rs

```diff
@@ -119,6 +119,7 @@ impl<'a> Parser<'a> {
     ///
     /// Options that are not valid unicode are transformed with replacement
     /// characters as by [`String::from_utf8_lossy`].
+    #[allow(clippy::should_implement_trait)]
```

Comment:
What is different between your clippy run and CI where these aren't being reported?